cookbook/import_export/create-connector.rst

```diff
@@ -42,20 +49,43 @@ The writer is implemented in the class ``Pim\Component\Connector\Writer\DummyIte
 
 We'll explain in next cookbook chapters how to create your own elements with real logic inside.
 
-.. note::
+.. warning::
 
-   It is also possible to create multiple job configuration files by placing them in a folder ``Resources/config/batch_jobs``. The job configurations must be yaml files of any name.
+   Please note that in version < 1.6, the file was always named "batch_jobs.yml" and was automatically loaded. The file content was very strict and was less evolutive than the more standard way to declare services that we use now.
 
-   For instance, I can declare a job configuration in a file named ``custom_jobs.yml`` like so ``Resources/config/batch_jobs/custom_jobs.yml``.
+Configure our JobParameters
+---------------------------
 
-Translate Job and Step titles
------------------------------
+To be executed, a Job is launched with a JobParameters which contains runtime parameters.
 
-Create a file ``Resources/translations/messages.en.yml`` in our Bundle to translate title keys.
+In our example, let's assume that our job needs a file path to write data, this file path must be provided and related to a writable directory to execute the job.
```

Comment:
needs a file path where to write data. This path must be provided and the directory has to be writable in order to execute the job.